phpBB/styles/prosilver/theme/stylesheet.css

```diff
@@ -20,3 +20,4 @@
 @import url("colours.css?v=3.3");
 @import url("responsive.css?v=3.3");
 @import url("tweaks.css?v=3.3");
+@import url("pm.css?v=3.3");
```

Comment:
You still have not fixed this... All colors need to be separated out into colours.css and this line needs to be moved above `@import url("colours.css?v=3.3");`